reactFirstProject/src/ProgressBar.jsx

```diff
@@ -0,0 +1,15 @@
+import './ProgressBar.css'
+export default function ProgressBar({totalScore, curQuestionId, questionsLength}){
```

Comment:
```suggestion
export default function ProgressBar({completion}){
```

Let's use only one prop here. Just a number from 0 to 100

Reply:
Oh, sure!